### PR TITLE
added create task table

### PR DESCRIPTION
**Pull request by KennethEnevoldsen**
Added CI for creating task table

### CODE REVIEW

File: .github/workflows/docs.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# GitHub action for the mmteb contributions
+# this way the CI related to the mmteb contributions will be separated from the linting CI
```
Comment (isaac-chung) [inline .github/workflows/docs.yml:2]:
```suggestion
# GitHub action for the task table generation.
```

File: docs/create_tasks_table.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    """Create (Authors, Year) from bibtex entry (author = {Authors}, year = {Year})"""
+    if bibtex is None:
+        return ""
+    # get authors from bibtex (author = {Authors} or author={Authors})
+    authors = re.search(r"author\s*=\s*{([^}]*)}", bibtex)
+    if authors is None:
+        return ""
+    authors = authors.group(1)
+    authors = [a.split(", ") for a in authors.split(" and ")]
+    # authors = [f"{a[0]} et al." if len(a) > 1 else a[0] for a in authors]
```
Comment (isaac-chung) [inline docs/create_tasks_table.py:19]:
```suggestion

```

File: .github/workflows/docs.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  create-table:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.RELEASE }}
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
```
Comment (isaac-chung) [inline .github/workflows/docs.yml:20]: Other workflow uses 3.10. Should we align at 3.8 or bump others to 3.10 as well?

**Review by isaac-chung — COMMENTED** [review]
Nice! A few small comments.


Should also merge main as there are some overlaps with https://github.com/embeddings-benchmark/mteb/pull/522.

File: .github/workflows/docs.yml (hunk 3 above)
Comment (KennethEnevoldsen) [reply .github/workflows/docs.yml:20]: I don't think it matters. I would just use the most stable version for everything except for tests (3.11 would probably also be fine)

**Pull request merged**